### ISSUE TITLE
Unable to use auto-play in bandcamp or skip to next

### ISSUE THREAD

**Issue by AlexN1997** [TeamNewPipe/NewPipe#5924]
### Checklist

- [x] I am using the latest version - 0.21.0 
- [ x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [ ] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x ] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
Go to bandcamp, select a video,select play in background, listen til the end.

### Screenshots/Screen recordings
![Capture+_2021-03-27-14-17-11~2](https://user-images.githubusercontent.com/76731207/112722341-36b15100-8f09-11eb-8ab1-efe9bca1693f.png)
Unable to select next video in bandcamp

### Device info

 - Android version/Custom ROM version: 8.0 (stock)
 - Device model: LG G5

**Comment by fynngodau** [comment]
Were you playing a track outside of a playlist? In this case, the reason is
https://github.com/TeamNewPipe/NewPipeExtractor/issues/593 and the issue was fixed by https://github.com/TeamNewPipe/NewPipeExtractor/pull/610 for version 0.21.2.

**Comment by AlexN1997** [comment]
> 
> 
> Were you playing a track outside of a playlist? In this case, the reason is
> [TeamNewPipe/NewPipeExtractor#593](https://github.com/TeamNewPipe/NewPipeExtractor/issues/593) and the issue was fixed by [TeamNewPipe/NewPipeExtractor#610](https://github.com/TeamNewPipe/NewPipeExtractor/pull/610) for version 0.21.2.

Yes, the song is outside of a playlist, but nothing changed (I'm on 0.21.2 now).
The same story applies:
Go to Bandcamp,
select a video (not a playlist),
select play in background (by tapping on the video or on the headphone),
listen to the end.
no next song

**Comment by fynngodau** [comment]
Right, there is an additional reason why this is not working, which I had forgotten about:

In the NewPipe app, `PlayerHelper.autoQueueOf` ignores items that are not `StreamInfoItems`. However bandcamp only suggests albums as recommended items, therefore there are no items that can be queued as next items.

Question: how would you expect the app to behave if auto-queue is enabled? Should it play the whole first recommended playlist?

**Comment by AlexN1997** [comment]
>
>Should it play the whole first recommended playlist?

I think that would be nice, kind of you to take a look at it!
